checkov/terraform/checks/resource/aws/DBInstanceLogging.py

```diff
@@ -0,0 +1,22 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class DBInstanceLogging(BaseResourceCheck):
```

Comment:
@gruebel I believe this can be a BaseResourceValueCheck

Reply:
it was before, but `BaseResourceValueCheck`  doesn't handle empty list correctly.